pkgs/development/python-modules/pytest-ansible/default.nix

```diff
@@ -1,46 +1,46 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, ansible
-, pytest
-, mock
+, ansible-core
+, coreutils
+, coverage
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "pytest-ansible";
-  version = "2.2.4";
+  version = "3.0.0";
 
   src = fetchFromGitHub {
     owner = "ansible";
-    repo = "pytest-ansible";
+    repo = pname;
     rev = "v${version}";
-    sha256 = "0vr015msciwzz20zplxalfmfx5hbg8rkf8vwjdg3z12fba8z8ks4";
+    sha256 = "sha256-kxOp7ScpIIzEbM4VQa+3ByHzkPS8pzdYq82rggF9Fpk=";
   };
 
-  patchPhase = ''
-    sed -i "s/'setuptools-markdown'//g" setup.py
+  postPatch = ''
+    substituteInPlace tests/conftest.py inventory --replace '/usr/bin/env' '${coreutils}/bin/env'
```

Comment:
Weird I thought `/usr/bin/env` was nix friendly but it is absent in the sandbox.